### ISSUE TITLE
artifact windows-ubuntu fails but report success

### ISSUE THREAD

**Issue by lukka** [Azure/azure-devtestlab#653]
created a Windows VM with this artifact [windows-ubuntu](https://github.com/Azure/azure-devtestlab/blob/c741a6703fef9a99a160901bd45bda17cc3fb066/Artifacts/windows-ubuntu/Artifactfile.json). When the machine was created, the execution reported succesful execution:

`````
Artifact Information
Artifact
Name
Windows Ubuntu
Repository
Public Artifact Repo
Status
Succeeded
Install Time
Wed May 13 2020 21:20:05 GMT-0700 (Pacific Daylight Time)
`````

Nevertheless, **it failed**. This is the output of the artifact:

````
Creating Ubuntu Distro Folder
Downloading Ubuntu Distro
Renaming .appx to .zip
Unzipping Distro
Installing Ubuntu
Microsoft Windows [Version 10.0.18363.836]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\Distros>C:\Distros\Ubuntu\ubuntu1804.exe install --root
Installing, this may take a few minutes...

WslRegisterDistribution failed with error: 0x8007019e

The Windows Subsystem for Linux optional component is not enabled. Please enable it and try again.

See https://aka.ms/wslinstall for details.
````

**Comment by leovms** [comment]
I researched and found that the following error indicates that the optional Windows feature called `Windows Subsystem for Linux (WSL)` may be disabled.
```
WslRegisterDistribution failed with error: 0x8007019e
```
Can you double check if this is the case?

**Comment by lukka** [comment]
yes, the Windows Feature is missing, but this is unrelated to the problem.

**Comment by leovms** [comment]
I realize that. I want to know if you are blocked by it. The problem lies in the way the install call is done. I suspect it is not bubbling up the exit code or throwing. We don't seem to check for that condition.

**Comment by lukka** [comment]
@leovms it is not a blocking problem

**Comment by lukka** [comment]
@leovms 

I verified on an Azure VM with Hyper-V and WSL enabled, the artifact cannot be applied successfully, it always fails with:

```
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x80070005
Error: 0x80070005 Access is denied.
```

It looks like the installation (i.e. the wsl.exe tool) must be run under an machine user, while the artifact scripts are running under NT_AUTHORITY\SYSTEM. The script does work indeed if i do run it manually from an admin powershell prompt.

If this is the case, the **windows-ubuntu** artifact should be removed as it is not working at all.

@benhillis FYI
